holo-key-manager-extension/scripts/helpers.ts

```diff
@@ -41,7 +41,7 @@ export const signMessageLogic = async ({ message, happId, session }: SignMessage
 	const pw = new TextEncoder().encode(session);
 	const keyUnlocked = cipherList[0].unlock(parseSecret(pw));
 
-	const appKey = keyUnlocked.derive(index);
+	const appKey = keyUnlocked.derive(index + DEEP_KEY_AGENT_OFFSET);
```

Comment:
I'm not a big fan of this "assume the order of the array is the appIndex". It seems very brittle.

I would've strongly preferred that we actually store the appIndex...but not going to push for this.

Reply:
I feel quite confident with this approach now, but maybe we can set some checks on the key-manager-store site to allow registering a new key only with the last appIndex+1?

And also on the get site to check if all appIndexes are returned so we are sure that there is no "hole"